config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -0,0 +1,20 @@
+[docker_cli/systemd]
+#: Systemd unit file ``sysd_unit_file``
+sysd_unit_file = p4321.service
+#: Systemd unit file destination ``sysd_unitf_dest``
+sysd_unitf_dest = /etc/systemd/system/p4321.service
+
+#: increase ``docker_timeout`` to 180 as building image can take longer
+docker_timeout = 300
+
+#: build image ``name``
+name=p4321
+
+#: image building args ``build_options_csv``
+build_options_csv = --force-rm,-t,%(name)s
+
+#: local ip address ``tcp_ip``
+tcp_ip = 127.0.0.1
+
+#: port to use ``tcp_port``
```

Comment:
Why do you have all these things like ``this`` and ``that`` in the comments? It seems like unnecessary duplication, and it takes away space from an actually helpful comment. But I've never seen that before, and am curious if it's just something I don't know about.